Source/Core/Core/Core.cpp

```diff
@@ -266,11 +272,13 @@ static void ResetRumble()
 // Called from GUI thread
 void Stop()  // - Hammertime!
 {
+  std::string game_id = SConfig::GetInstance().GetGameID();
```

Comment:
You could move this further down, next to your call of `AddTime`; and potentially even make it `const`.